pubsub/gcloud/aio/pubsub/subscriber.py

```diff
@@ -224,7 +224,9 @@ async def producer(
                 new_messages = []
                 try:
                     new_messages = await subscriber_client.pull(
-                        subscription=subscription, max_messages=max_messages)
+                        subscription=subscription,
+                        max_messages=max_messages,
+                        timeout=30)
```

Comment:
Could you document your tribal knowledge here? Otherwise this 30 second timeout will look like an arbitrarily-selected value to those who wish to modify this in the future.

Reply:
a comment will do or you mean something more solid?

Reply:
A comment would be fine 🙂